tap_dashboard_reports/period.py

```diff
@@ -50,6 +51,28 @@ def _calculate_start_date(self):
         return self._end_date - relativedelta(days=self._periods)
 
 
+class PeriodWeek(PeriodDefault):
+    def iterate(self):
+        cursor = self._start_date
+        while cursor < self._end_date:
+            interval_start = cursor
```

Comment:
Start date is not Monday here

Reply:
return 
```
def first_day_of_week(dt):
    return dt - timedelta(days=dt.weekday())
```

always returns mondays of the week